controllers/drplacementcontrol_controller_test.go

```diff
@@ -595,7 +597,7 @@ func verifyVRGManifestWorkCreatedAsPrimary(managedCluster string) {
 		return err == nil
 	}, timeout, interval).Should(BeTrue())
 
-	Expect(len(createdVRGRolesManifest.Spec.Workload.Manifests)).To(Equal(2))
+	Expect(len(createdVRGRolesManifest.Spec.Workload.Manifests)).To(BeNumerically(">=", 2))
```

Comment:
The vrg and pvc cluster roles and role bindings are now in the same manifestwork (to simplify management since they have the same lifetime) for a total of 4 manifests.

Reply:
So, test against 4 then.

Reply:
I think more manifests are ok.  This particular test applies only the first two manifests and then verifies a vrg can be created.